snapcraft/internal/repo/errors.py

```diff
@@ -81,6 +81,16 @@ def __init__(self, package):
         super().__init__(package=package)
 
 
+class SnapUnavailableError(RepoError):
+
+    fmt = ('Failed to install or refresh a snap: {snap_name!r} does not exist '
+           'or is not available on the desired channel {snap_channel!r}. '
+           'Verify that the snap is available.')
```

Comment:
Perhaps "use `snap info {snap_name}` to list available channels" or something that similarly directs them to how to verify that it's available?